project-reports/2024/2024-Q3-Hyperledger-Bevel.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Bevel
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# 2024 Q3 Hyperledger Bevel
+
+# Project Health
+
+[Hyperledger Bevel LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
+
+
+1. Getting code contributions for Hyperledger Bevel still remains a challenge.
+2. The Bevel project has received no external contributions; nearly all contributions for the last couple of releases have come from the Accenture team. Accenture will not be able to maintain the project without other organization or individual contributor help in maintaining.
```

Comment:
Would you need additional assistance on this topic?

Reply:
Yes, please. Can connect to discuss

Reply:
does this summary on external contributions take into account the `bevel-fabric-operator` component? Isn't that contributed by a different organization? or is that not considered under the Bevel umbrella, which if so would be a mistake.

Reply:
the bevel and bevel-fabric-operator have separate code base/repo which runs independently, the comment here is for bevel

Reply:
Would having an umbrella of projects help in reaching the goal? For instance, Cello is another project with kubernetes operators.